vsdownload.py

```diff
@@ -626,7 +639,26 @@ def moveVCSDK(unpack, dest):
 
     socket.setdefaulttimeout(15)
 
-    packages = getPackages(getManifest(args))
+    if args.host_arch is None:
+        args.host_arch = platform.machine().lower()
+        if platform.system() == 'Darwin':
+            # There is no prebuilt arm64 Wine on macOS.
+            args.host_arch = 'x64'
```

Comment:
Sorry to ask about this now, not having noted it before... We're using `""` for string literals in this script so far, almost consistently, so I'd appreciate if you'd match that style for the newly added code too.

Reply:
@mstorsjo i already fixed this in the PR i sent. This is what I was talking about normalizing the quotes

Reply:
I know. But different things belong in different commits. You should not mix in unrelated cleanups in a functional change.

Is this already has been merged to master, I would have pushed a commit to unify it, but as this is in a yet unmerged PR, I prefer fixing up within this PR before merging it.

Reply:
Got it. Once this is merged, I'll rebase and update the PR and include the changes you suggested

Reply:
Any plans to merge this sooner https://github.com/mstorsjo/msvc-wine/pull/144

This is very nice feature and has reduced the size of the container drastically.

Reply:
> Sorry to ask about this now, not having noted it before... We're using `""` for string literals in this script so far, almost consistently, so I'd appreciate if you'd match that style for the newly added code too.

Done.